project/external-trusted-reviewers.md

```diff
@@ -5,3 +5,5 @@ Mbed TLS welcomes contributors from the wider community who would be interested
 We are grateful for the contribution of our current trusted reviewers:
 
```

Comment:
Preexisting: is there a reason we don't list all [members of the reviewers team](https://github.com/orgs/Mbed-TLS/teams/mbed-tls-reviewers/members) here?